From 3dfb3db2950f57f2a2e0330cd480d7880d314729 Mon Sep 17 00:00:00 2001
From: Jiri Denemark <jdenemar@redhat.com>
Date: Wed, 3 Sep 2014 10:51:14 +0200
Subject: [PATCH] spec: Fix preun script for daemon

%systemd_preun macro cannot be split into several lines.

https://bugzilla.redhat.com/show_bug.cgi?id=1136736

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
(cherry picked from commit 6209454d5b10c2d89faab8a56b250f1569d823fe)
---
 libvirt.spec.in | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/libvirt.spec.in b/libvirt.spec.in
index 0741c73..f408c64 100644
--- a/libvirt.spec.in
+++ b/libvirt.spec.in
@@ -1677,11 +1677,7 @@ fi
 %preun daemon
     %if %{with_systemd}
         %if %{with_systemd_macros}
-            %systemd_preun \
-                libvirtd.socket \
-                libvirtd.service \
-                virtlockd.socket \
-                virtlockd.service
+            %systemd_preun libvirtd.socket libvirtd.service virtlockd.socket virtlockd.service
         %else
 if [ $1 -eq 0 ] ; then
     # Package removal, not upgrade
